src/map/script.cpp

```diff
@@ -27123,6 +27123,54 @@ BUILDIN_FUNC(opentips){
 #endif
 }
 
+BUILDIN_FUNC(setdialogalign){
```

Comment:
Applies to all script functions:
Would be nice to validate the input and report invalid usages.